web/webapp/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "prebuild": "chmod +x ./copy-circuits.sh && sh ./copy-circuits.sh",
+    "prebuild": "chmod +x ./copy-circuits.sh && sh ./copy-circuits.sh && chmod +x ./copy-env.sh && sh ./copy-env.sh",
```

Comment:
I didn't notice these `chmod`s before. You don't need them. Just do `chmod +x` locally and `git add` these files. Git is keeping track of executable bits of files.